lib/web/apiserver_test.go

```diff
@@ -862,8 +862,10 @@ func TestClusterNodesGet(t *testing.T) {
 
 	endpoint := pack.clt.Endpoint("webapi", "sites", clusterName, "nodes")
 
+	query := url.Values{"sort": []string{"name"}}
```

Comment:
Not requiring things to be sorted may reduce database stress
Also, for this particular check seems to be unnecessary (we only care if exists one or none)

Reply:
it's related to this comment https://github.com/gravitational/teleport/pull/15637#discussion_r952105198, the tests that include the `sort` query params, has a test that tested for the `totalCount` which can only be returned with calls to ListResources that triggers fake pagination (sort being one of them)